### ISSUE TITLE
[a11y][Popover]: Focus moves out of popover items

### ISSUE THREAD

**Issue by ArtemJJS** [SAP/ui5-webcomponents#11341]
### Bug Description

We have an issue with popover on a real iPhone device with an enabled VoiceOver screen reader and external keyboard connected. Focus is not correctly ordered, navigation goes not through items placed inside the popover (SideNavigation), but for external items.

We are using Popover containing SideNavigation inside for mobile devices. Popover is inserted on the page using the createPortal option.

There are no issues with VoiceOver enabled on a MacBook, on an iPhone without VoiceOver, or on any other device setup. The only case where it is reproducible is a real iPhone device with VoiceOver enabled when navigation is performed from an external Bluetooth keyboard.

I also not completely sure which component causes this behavior, Popover or SideNavigation.

### Affected Component

Popover or SideNavigation

### Expected Behaviour

Navigation goes through the side navigation items first on the iPhone when VoiceOver is enabled. 

### Log Output, Stack Trace or Screenshots

https://github.com/user-attachments/assets/8bca997e-7688-481a-bb79-0fb419adf9f8

### Priority

High

### UI5 Web Components Version

2.0.8

### Browser

Safari

### Operating System

MacOS

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Valkanister** [comment]
Hello @ArtemJJS , 

The scenario you are describing is dependent on multiple products and interactions which are outside of the scope of the UI5 web components. Especially the interoperability of VoiceOver with an external keyboard. Is there another scenario where the issue is reproducible? 

Kind regards, 

Georgi

SAP Development Support

**Comment by ArtemJJS** [comment]
@Valkanister no, we have this issue for VoiceOver and external keyboard on mobile devices only. For other cases, it works well. Do you have any ideas on how to deal with that? Probably one of the developers worked on this component can take a look and recommend anything to try?

**Comment by TeodorTaushanov** [comment]
Hello @SAP/ui5-webcomponents-topic-rd, can you check this issue?

**Comment by TeodorTaushanov** [comment]
Hello @ArtemJJS,

We cannot control where the focus goes in this particular case. UI5 Web Components provide the prerequisites for screen reader support based on the HTML, ARIA, and WCAG standards. All screen readers that follow those standards should work fine. Nevertheless, there are deviations in the interpretation depending on the combination of device, browser and screen reader. 

Best,
Teodor
